frame/benchmarking/src/lib.rs

```diff
@@ -1930,3 +1930,5 @@ macro_rules! define_benchmarks {
 		}
 	}
 }
+
+pub use frame_support::benchmarking::*;
```

Comment:
Can we please import `frame-support-procedural` directly here and move the entire `benchmarking` module here? I don't get why we need this in `frame-support`

Reply:
oh that is even better, I'll do that

Reply:
another approach would be to rename `#[benchmarks]` to `#[benchmarks_module]`, then nothing would collide cc @ggwpez

Reply:
note that my half-measure of just doing a wildcard import _does_ seem to work (current diff) except for in the UI tests -- in the balances and message queue pallet, I only need to do `use frame_benchmarking::*;` and it knows I mean the attribute not the tt macro because I did a `pub use frame_support::benchmarking::*;` in `frame_benchmarking`. This is a hacky solution, though, imo.

Reply:
I think `frame_benchmarking::v2::*` is not a horrible idea, given that otherwise `v1`/`v2` are mixed and we risk future collisions.

Reply:
Ok I'm fine with this as well. Should we premptively export the current stuff under a `v1` module as well so later down the road we can deprecate not doing that?

Reply:
Yea sounds good, unless @bkchr has a better idea :stuck_out_tongue:

Reply:
No better idea :P